packages/iTwinUI-react/src/core/ComboBox/ComboBox.test.tsx

```diff
@@ -209,77 +210,99 @@ it('should handle keyboard navigation', () => {
   const mockOnChange = jest.fn();
   const { container } = renderComponent({ id, onChange: mockOnChange });
 
+  userEvent.tab();
+
   const input = assertBaseElement(container);
-  input.focus();
   expect(input).toHaveAttribute('aria-controls', `${id}-list`);
 
-  const items = document.querySelectorAll('.iui-menu-item');
+  let items = document.querySelectorAll('.iui-menu-item');
```

Comment:
I don't understand this change and why `items` needs to be redefined every time

Reply:
the test was failing and when I tried debugging, I saw that `items` was referring to the old instance, so it had to be queried again. i have no clue how/why it was working before, but this change was just to get the test to pass.